### PR TITLE
Add hiibot_iots2.

### PR DESCRIPTION
**Pull request by BradChan**

### CODE REVIEW

File: ports/esp32s2/boards/hiibot_iots2/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,17 @@
+USB_VID = 0x239A
+USB_PID = 0x80E8
```
Comment (tannewt) [inline ports/esp32s2/boards/hiibot_iots2/mpconfigboard.mk:2]: 0x239A is Adafruit's USB VID. Please change this to one you are allowed to use. If the board is open source, then you can get one from https://pid.codes.

**Review by tannewt — CHANGES_REQUESTED** [review]
Please fix pre-commit as well as getting your own USB VID/PID. You can automatically fix pre-commit errors by installing it locally: https://pre-commit.com/#installation

**Comment by ladyada** [comment]
@BradChan please get a PID from espressif here https://github.com/espressif/usb-pids

**Comment by dhalbert** [comment]
@BradChan did you get another PID?

**Comment by dhalbert** [comment]
@BradChan Would you like to finish this off? DId you get a PID?

**Comment by dhalbert** [comment]
@BradChan Closing for now. Please reopen when you're ready. Thanks!

**Pull request closed without merging**